cookbook/knowledge/embedders/mistral_embedder.py

```diff
@@ -12,12 +12,13 @@
 print(f"Embeddings: {embeddings[:5]}")
 print(f"Dimensions: {len(embeddings)}")
 
-# Example usage:
+# Example usage with batch embedding enabled:
 knowledge = Knowledge(
     vector_db=PgVector(
         db_url="postgresql+psycopg://ai:ai@localhost:5532/ai",
         table_name="mistral_embeddings",
         embedder=MistralEmbedder(),
+        use_batch=True,  # Enable batch embedding to reduce API calls
```

Comment:
I know this is the existing attribute, but it sucks as a name. And it doesn't currently work, so it is safe to update I think.
`enable_batch_embeddings` would be betteR?